lib/simulator-xcode-9.js

```diff
@@ -51,8 +52,14 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   }
 
   async shutdown () {
+    await restoreTouchEnrollShortcuts();
```

Comment:
here one must verify whether UI client is running first, because it might be we are running in headless mode